mycroft/configuration/__init__.py

```diff
@@ -98,10 +98,20 @@ def load(config=None, locations=None, keep_user_config=True):
         return config
 
     @staticmethod
+    def merge_conf(base, delta):
```

Comment:
Would you mind adding a short docstring here explaining the function.

Something along the lines

```python
    """
        Class method recursively merging configuration dictionaries.

        Args:
            base:  Target for merge
            delta: Dictionary to merge into base
    """